crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/initiate.rs

```diff
@@ -108,12 +140,31 @@ impl ShardedGossipLocal {
 
         let mut gossip = Vec::with_capacity(3);
 
+        let mut agent_list = Vec::new();
+        for agent in local_agents.iter() {
+            if let Ok(Some(info)) = self
+                .evt_sender
+                .get_agent_info_signed(GetAgentInfoSignedEvt {
```

Comment:
Again I would do this a one call as it's all the local agents. Although I guess it depends if we are optimizing for the holo usecase or the one agent per space per conductor?

Reply:
:+1: updated